packages/client/components/DashboardAvatars/DashboardAvatars.tsx

```diff
@@ -1,94 +1,112 @@
 import styled from '@emotion/styled'
 import graphql from 'babel-plugin-relay/macro'
-import React, {useRef} from 'react'
-import {createFragmentContainer} from 'react-relay'
-import {Breakpoint} from '~/types/constEnums'
-import makeMinWidthMediaQuery from '~/utils/makeMinWidthMediaQuery'
-import useAvatarsOverflow from '../../hooks/useAvatarsOverflow'
+import React from 'react'
+import {commitLocalUpdate, createFragmentContainer} from 'react-relay'
+import {Breakpoint, ElementWidth} from '~/types/constEnums'
+import useAtmosphere from '../../hooks/useAtmosphere'
+import useBreakpoint from '../../hooks/useBreakpoint'
+import useMutationProps from '../../hooks/useMutationProps'
+import ToggleManageTeamMutation from '../../mutations/ToggleManageTeamMutation'
 import {PALETTE} from '../../styles/paletteV3'
 import {DashboardAvatars_team} from '../../__generated__/DashboardAvatars_team.graphql'
-import AddTeamMemberAvatarButton from '../AddTeamMemberAvatarButton'
 import ErrorBoundary from '../ErrorBoundary'
 import DashboardAvatar from './DashboardAvatar'
 
-const desktopBreakpoint = makeMinWidthMediaQuery(Breakpoint.SIDEBAR_LEFT)
-
 const AvatarsList = styled('div')({
   display: 'flex',
-  justifyContent: 'flex-start',
-  marginTop: 16,
-  width: '75%',
-  [desktopBreakpoint]: {
-    justifyContent: 'flex-end',
-    marginTop: 0,
-    width: '100%'
-  }
+  flexDirection: 'column',
+  marginRight: 6
 })
 
-const ScrollContainer = styled('div')({
+const Wrapper = styled('div')({
   display: 'flex',
-  maxWidth: '100%',
-  overflow: 'hidden'
-})
-
-const ItemBlock = styled('div')({
-  marginRight: 8,
+  justifyContent: 'center',
   position: 'relative',
-  [desktopBreakpoint]: {
-    marginBottom: 8,
-    marginLeft: 8,
-    marginRight: 0
-  }
+  left: -4
 })
 
-interface Props {
-  team: DashboardAvatars_team
-}
+const OverflowWrapper = styled('div')({
+  width: ElementWidth.DASHBOARD_AVATAR_OVERLAPPED
+})
 
 const OverflowCount = styled('div')({
   alignItems: 'center',
+  border: `2px solid ${PALETTE.SLATE_200}`,
   backgroundColor: PALETTE.SKY_400,
   borderRadius: '50%',
   display: 'flex',
-  height: 32,
+  height: 28,
   justifyContent: 'center',
   color: '#fff',
   fontSize: 12,
   fontWeight: 600,
   overflow: 'hidden',
   userSelect: 'none',
-  width: 32
+  width: ElementWidth.DASHBOARD_AVATAR,
+  '&:hover': {
+    cursor: 'pointer'
+  }
+})
+
+const Label = styled('div')({
+  fontSize: 12,
+  height: 12,
+  lineHeight: 1,
+  fontWeight: 600,
+  color: PALETTE.SLATE_700,
+  textAlign: 'center',
+  width: '100%',
+  '&:hover': {
+    cursor: 'pointer'
+  }
 })
 
+interface Props {
+  team: DashboardAvatars_team
+}
+
 const DashboardAvatars = (props: Props) => {
   const {team} = props
-  const {id: teamId, isLead: isViewerLead, teamMembers} = team
-  const wrapperRef = useRef<HTMLDivElement>(null)
-  const avatarsRef = useRef<HTMLDivElement>(null)
-  const maxAvatars = useAvatarsOverflow(wrapperRef, avatarsRef)
```

Comment:
Now that we have a limit of 10 avatars, I don't think we need to dynamically count the max number of avatars with `useAvatarsOverflow`. On mobile, we drop from 10 to 6 avatars. We could use `useAvatarsOverflow` to see if there's space for 1 or 2 more avatars on mobile, i.e. show 7/8 avatars, but I think the cost of the extra code outweighs the benefit of 1/2 more avatars on mobile

Reply:
That works for me in this case